application/backup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+for dbName in ${DB_NAMES}; do
+    logger -p user.info "backing up ${dbName}..."
+
+    start=$(date +%s)
+    runny $(mysqldump -h ${MYSQL_HOST} -u ${MYSQL_USER} -p"${MYSQL_PASSWORD}" ${dbName} > /tmp/${dbName}.sql)
```

Comment:
You could include the gzip here, instead of making it a separate step ...
runny $(mysqldump -h ${MYSQL_HOST} -u ${MYSQL_USER} -p"${MYSQL_PASSWORD}" ${dbName} | gzip > /tmp/${dbName}.sql)

Reply:
if you don't mind, I think I like the separate steps for the sake of logging in `runny`